BuildAndTest.proj

```diff
@@ -1,7 +1,7 @@
 <Project
   xmlns="http://schemas.microsoft.com/developer/msbuild/2003"
   DefaultTargets="BuildAndTest"
-  ToolsVersion="12.0">
+ >
```

Comment:
I thought that csproj.dll used the existence of this to trigger project upgrades from .NET 1.0 -> 2.0 or 3.5.